great-idea/index.html

```diff
@@ -17,7 +17,69 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
-      
+    <header>
+      <nav>
+        <a href ="services.html"> Services</a>
+        <a href =""> Product</a>
+        <a href =""> Vision</a>
+        <a href =""> Features</a>
+        <a href =""> About</a>
+        <a href =""> Contact</a>
+        <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </nav>
+
+    <div class="container">  
+      <div class="phrase">
+        <h1>Innovation On Demand</h1>
+        <div class="button">
+          <p>Get Started</p>
+        </div>
```

Comment:
i definitely prefer using divs to just create my own buttons, but you'll have to face styling over default buttons eventually. but i love this one!